forward_engineering/helpers/columnDefinitionHelper.js

```diff
@@ -1,8 +1,7 @@
-const _ = require('lodash');
-const templates = require('../configs/templates');
-
 module.exports = app => {
+	const templates = require('../configs/templates');
 	const assignTemplates = app.require('@hackolade/ddl-fe-utils').assignTemplates;
+	const _ = app.require('lodash');
```

Comment:
@Nightlngale The idea was to progressively to remove providing lodash from the app not to extend its usage.  Isn't it a better way to use the installed version from the plugin?

Reply:
Especially Lodash is part of the deps: https://github.com/hackolade/Snowflake/blob/ac4733986000d7a95e1745e6362b82640a741ffe/package.json#L56